src/core/js/helpers.js

```diff
@@ -196,21 +230,158 @@ define([
+            return new Handlebars.SafeString('<div class="aria-label" role="img" aria-label="'+values.join(' ')+'"></div>');
+        },
+
+        /**
+         * Returns an `a` tag which when receiving focus causes the focus to wrap
+         * to the top of the readable document.
+         *
+         * @returns {string}
+         */
+        a11y_wrap_focus: function() {
+            var cfg = Adapt.config.get('_accessibility');
+            if (cfg._isPopupWrapFocusEnabled === false) return '';
+            return new Handlebars.SafeString('<a class="a11y-focusguard a11y-ignore a11y-ignore-focus" role="presentation">&nbsp;</a>');
+        },
+
+        /**
+         * Creates the attributes for a subject heading text. `role="heading"` and
+         * `aria-level="#"`. It will use the `_ariaLevel` attribute from the current
+         * context if specified, a number if given as the `levelOrType` parameter,
+         * or a name from the configured aria levels hash.
+         *
+         * @param {number|string} levelOrType
+         * @returns {string}
+         */
+        a11y_attrs_heading: function(levelOrType) {
+            // get the global configuration from config.json
+            var cfg = Adapt.config.get('_accessibility');
+            // default level to use if nothing overrides it
+            var level = 1;
+
+            // first check to see if the Handlebars context has an override
+            if (this._ariaLevel) {
+                levelOrType = this._ariaLevel;
+            }
+
+            if (isNaN(levelOrType) === false) {
+                // if a number is passed just use this
+                level = levelOrType;
+            } else if (_.isString(levelOrType)) {
+                // if a string is passed check if it is defined in global configuration
+                cfg._ariaLevels = cfg._ariaLevels || defaultAriaLevels;
+                if (cfg._ariaLevels && cfg._ariaLevels['_'+levelOrType] !== undefined) {
+                    level = cfg._ariaLevels['_'+levelOrType];
+                }
+            }
+
+            return new Handlebars.SafeString(' role="heading" aria-level="'+level+'" ');
+        },
+
+        a11y_attrs_tabbable: function() {
+            Adapt.a11y.log.deprecated('a11y_attrs_tabbable should not be used. tabbable elements should be natively tabbable.');
+            return new Handlebars.SafeString(' role="region" tabindex="0" ');
         }
 
     };
 
     // Compatibility references
-    helpers['if_value_equals'] = helpers['equals'];
-    helpers['numbers'] = helpers['inc'];
-    helpers['lowerCase'] = helpers['lowercase'];
+    _.extend(helpers, {
 
-    for (var name in helpers) {
-        if (helpers.hasOwnProperty(name)) {
-             Handlebars.registerHelper(name, helpers[name]);
+        if_value_equals: function() {
+            Adapt.a11y.log.deprecated('if_value_equals, use equals instead.');
+            return helpers['equals'].apply(this, arguments);
```

Comment:
I don't think there's any need to use 'associative array' style lookup here (or on L372 and L377)
```suggestion
            return helpers.equals.apply(this, arguments);
```